.github/workflows/test.yml

```diff
@@ -50,13 +50,17 @@ jobs:
           make -C app/controlplane migration_sync
 
       # Check that the generated ent code is up to date
-      # see https://entgo.io/docs/ci/
-      - uses: ent/contrib/ci@e38dfb6484dfbe64b8bd060fe6a219a1aa5da770 # master
-        name: "Check all ent generated code is checked in"
+      - name: "Check all ent generated code is checked in"
         if: ${{ matrix.app != 'main-module' }}
```

Comment:
I think the solution is adding here `matrix.app != 'cli'` as well. We don't use Ent in the CLI